chalk-ir/src/debug.rs

```diff
@@ -21,6 +21,18 @@ impl<I: Interner> Debug for AssocTypeId<I> {
     }
 }
 
+impl<I: Interner> Debug for Ty<I> {
+    fn fmt(&self, fmt: &mut Formatter<'_>) -> Result<(), Error> {
+        I::debug_ty(self, fmt).unwrap_or_else(|| unimplemented!())
+    }
+}
+
+impl<I: Interner> Debug for Lifetime<I> {
+    fn fmt(&self, fmt: &mut Formatter<'_>) -> Result<(), Error> {
+        I::debug_lifetime(self, fmt).unwrap_or_else(|| unimplemented!())
```

Comment:
This is triggering a failure in one of the tests, presumably because the test does not setup the TLS, but the test also doesn't instantiate a `Program` so I'm not sure how best to go about fixing it.